src/zepben/eas/client/work_package.py

```diff
@@ -127,6 +129,78 @@ class MeterPlacementConfig:
     """The ID of the meter group to use for populating EnergyMeters at EnergyConsumers."""
 
 
+@dataclass
+class FixedTimeLoadOverride:
+
+    load_watts_override: Optional[float]
+    """
+    The reading to be used to override load watts
+    """
+
+    gen_watts_override: Optional[float]
+    """
+    The reading to be used to override gen watts
+    """
+
+    load_var_override: Optional[float]
+    """
+    The reading to be used to override load var
+    """
+
+    gen_var_override: Optional[float]
+    """
+    The reading to be used to override gen var
+    """
+
+
+@dataclass
+class TimePeriodLoadOverride:
+
+    load_watts_override: Optional[List[float]]
```

Comment:
Same thing as the other class, remove the override suffix.

Reply:
removed too